cypress/e2e/Search/search_audios_spec.js

```diff
@@ -6,14 +6,14 @@
  *
  */
 
-import { setToken } from '../../support/e2e';
-
+ import { setToken } from '../../support/e2e';
+ 
 describe('Search audios', () => {
   beforeEach(() => {
     setToken();
     cy.apiroute('GET', '/draft-api/v1/drafts/licenses/', 'licenses');
     cy.apiroute('GET', '/taxonomy/v1/subjects?language=nb', 'allSubjects');
-    cy.apiroute('GET', '/audio-api/v1/audio/?page=1&page-size=10&sort=-relevance', 'searchAudios');
+    cy.apiroute('GET', '/audio-api/v1/audio/?exclude-revision-log=false&fallback=false&include-other-statuses=false&page=1&page-size=10&sort=-relevance*', 'searchAudios');
```

Comment:
Det ser ut som om du har fått med en del parameter som kun gir meining for artikler.

Reply:
Når man så på nettverkskallet til testen så var det der hva den kalte `get` mot. Om ikke det matcha så kjørte ikke testen igjennom